### ISSUE TITLE
Offckb always displays a deprecation warning.

### ISSUE THREAD

**Issue by jordanmack** [RetricSu/offckb#47]
The offckb tool always displays the following warning:

```
(node:12542) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead
```

We need to find a way to remove or suppress this warning so new developers are not distracted by it.

This was observed using Node v20.12.0 LTS (newest LTS via nvm).

**Comment by RetricSu** [comment]
-https://github.com/RetricSu/offckb/pull/50
not sure if this will work

**Comment by jordanmack** [comment]
Yes, this is fixed in v0.1.0.
